docker/snaptools/makeCompliantBAM.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import pysam
+import argparse
+import sys
+import os
+
+def main():
+    parser = argparse.ArgumentParser(description="Make a BAM file with cellular barcodes in the "
+                                                 + "read names compliant by moving them to the CB tag")
+    parser.add_argument('--input-bam', dest="inputbam", help="input bam file")
+    parser.add_argument('--output-bam', dest="outputbam", help="output bam file")
+
+    args = parser.parse_args()
+
+    def checkArgs(args):
+        if args.inputbam is None:
+            sys.exit("Input BAM is not defined")
+        if args.outputbam is None:
+            sys.exit("Output BAM is not defined")
+        if not os.path.isfile(args.inputbam):
+            sys.exit("Input BAM is not a file")
+
+    checkArgs(args);
+
+    inputbamfilename = "input.bam"
+    bamfile = pysam.AlignmentFile(args.inputbam, 'rb')
+
+    outputfilename = "output.bam"
+    outbam = pysam.AlignmentFile(args.outputbam, 'wb', template=bamfile)
+
+    counter = 0
+
+    for read in bamfile:
+        counter += 1
+        if (counter % 100000 == 0):
+            print counter
```

Comment:
Whoa, python 2.7? This should be specified in the header if that's the case; this script will fail on python 3. 

```bash
#!/usr/bin/env python2.7
```